Assets/TriggerTaskMode.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class TriggerTaskMode : MonoBehaviour
+{
+
+    [SerializeField]
+    private string shootingScene, extinguishingScene, craneOperatingScene;
+
+    private void OnTriggerEnter(Collider other)
+    {
+        print(other.gameObject.name);
+        if(other.gameObject.name == "ShootingTask")
+        {
+            SceneManager.LoadScene(shootingScene.ToString(), LoadSceneMode.Single);
+        }
+        else if(other.gameObject.name == "ExtinguishingTask")
+        {
+            SceneManager.LoadScene(extinguishingScene.ToString(), LoadSceneMode.Single);
+        }
+        else if(other.gameObject.name == "CraneOperatingTask")
+        {
+            SceneManager.LoadScene(craneOperatingScene.ToString(), LoadSceneMode.Single);
+        }
```

Comment:
Wanneer je merkt dat je veel "if else" gebruikt, kan je kijken of een switch case iets is.

[Switch case](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch)

Reply:
Ook zie ik dat je de namen van de scenes hard-code, je kan hier heel mooi property classes voor gebruiken

[Property class](https://stackoverflow.com/questions/630803/associating-enums-with-strings-in-c-sharp)